### DISCUSSION TITLE
How do I convert LocalDate to String?

### DISCUSSION THREAD

**Discussion by amartinezp97** [community/community#107466] [Repositories] [answered]
### Select Topic Area

Question

### Body

I need to transform the value of LocalDate to a String. How can I do this?

**Comment by mmascheroni** [comment, marked as answer]
Hello! Here, I'm providing an example code for you on Java: 

LocalDate localDate = LocalDate.now();
DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd");
String fechaIngresoString = localDate.format(formatter);

On Javascript:

let currentDate = new Date();
let options = { year: 'numeric', month: '2-digit', day: '2-digit' };
let fechaIngresoString = currentDate.toLocaleDateString('en-US', options);
